packages/core/src/decorators/controller.decorator.ts

```diff
@@ -14,5 +14,10 @@ export function Controller(routePrefix = ""): ClassDecorator {
     if (!Reflect.hasMetadata(MetadataKeys.ROUTES, target)) {
       Reflect.defineMetadata(MetadataKeys.ROUTES, [], target);
     }
+
+    // Automatically setup dependency metadata here so users don't need to add @Injectable to controllers
+    const dependencyKey = Symbol.for(target.toString());
```

Comment:
Add ability for users to set custom identifier via `@Controller` like you can do with `@Injectable`